plugins/imgur_a.js

```diff
@@ -42,32 +46,41 @@ hoverZoomPlugins.push({
                     
                     switch (view) {
                         case 'signin':
-                            return;
+                        return;
                         case 'a': // album view:
                         case 'gallery':
-                            var anchor = matches[3];
-                            data.hoverZoomGallerySrc = [];
-                            data.hoverZoomGalleryCaption = [];
 
                             // Future alternative: https://imgur.com/ajaxalbums/getimages/{hash}/hit.json?all=true
                             var albumUrl = 'https://api.imgur.com/3/album/' + hash + '.json';
                             $.ajax(albumUrl, {headers: {"Authorization": "Client-ID 1d8d9b36339e0e2"}}).done(function (imgur) {
                                 if (imgur.error) {
                                     data.hoverZoomSrc = createUrls(hash);
                                     res.push(link);
-                                } else {
+                                } 
+                                //todo: Possible faster implementation later on for single images - leave commented for now
```

Comment:
Remove this commented out code. Gallery optimization should be done in common code, not in plugins.